src/Raven.Server/Web/System/OngoingTasksHandler.cs

```diff
@@ -318,6 +331,37 @@ private static IEnumerable<OngoingTask> CollectEtlTasks(DatabaseRecord databaseR
             }
         }
 
+        private static string GetDatabase(string name, ServerStore store, out DocumentDatabase db)
+        {
+            string error = null;
+            db = null;
+            try
+            {
+                if (store.DatabasesLandlord.DatabasesCache.TryGetValue(name, out var task))
+                {
+                    if (task == null)
+                    {
+                        throw new DatabaseLoadFailureException("Database task is 'null'.");
```

Comment:
Which database task?

Reply:
those errors will be caught by the 'catch' and add the database name to the error